Assets/Scripts/HealthBar.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+/*
+ * UI player health bar
+ * Displays player health remaining using a heart tally
+ */
+public class HealthBar : MonoBehaviour
+{
+    public Robo player;
+    public Heart[] hearts;
```

Comment:
Where is this initialized?

Reply:
The HealthBar, Robo, and Heart[] elements are all initialized within the scene editor and I hook them up together using the Inspector. Same as how we hook up the Duncan prefab with the duncan GameObject variable in the inspector for Robo.

Reply:
It's going to require a bit of experimentation. I'll get back to you on this.